outscale/data_source_outscale_public_catalog.go

```diff
@@ -0,0 +1,125 @@
+package outscale
+
+import (
+	"context"
+	"time"
+
+	oscgo "github.com/outscale/osc-sdk-go/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-outscale/utils"
+)
+
+func dataSourceOutscaleOAPIPublicCatalog() *schema.Resource {
```

Comment:
```ReadPublicCatalog``` return too many elements to be a singular data source.
Can you rename  ```dataSourceOutscaleOAPIPublicCatalog```  to ``` dataSourceOutscaleOAPIPublicCatalogs ```and the filename to  ```data_source_outscale_public_catalogs.go``` please?

Reply:
It is only one catalog, with many entries no ?